drape_frontend/user_mark_shapes.cpp

```diff
@@ -560,12 +545,7 @@ void CacheUserLines(ref_ptr<dp::GraphicsContext> context, TileKey const & tileKe
 
   double const vs = df::VisualParams::Instance().GetVisualScale();
```

Comment:
Visual scale can be changed in runtime depending on CarPlay or AA screen in the future, right?